pkg/reporegistry/repository.go

```diff
@@ -58,7 +59,7 @@ func LoadAllRepositories(confPaths []string) (rpmmd.DistrosRepoConfigs, error) {
 					return nil, err
 				}
 
-				log.Println("Loaded repository configuration file:", configFile)
+				logrus.Info("Loaded repository configuration file: %s", configFile)
```

Comment:
This should be `Infof()`, not `Info()`.

Reply:
Silly me, thanks for spotting! I am a bit short on time but ideally I would have added a test for this too /o\